docs/src/userguide/language_basics.rst

```diff
@@ -88,7 +88,10 @@ define global C variables.
 
             def func():
                 cdef int i, j, k
-                cdef float f, g[42], *h
+                cdef float f
+                cdef float[42] g
+                cdef float *h
+                # cdef float f, g[42], *h  # mix of pointers, arrays and values in a single line is deprecated
 
                 i = j = 5
 
```

Comment:
```suggestion

        Cython currently supports two ways to declare an array:

        .. code-blobk:: cython

            cdef int arr1[4], arr2[4]  # C style array declaration
            cdef int[4] arr1, arr2     # Java style array declaration

        Both of them generate the same C code, but the Java style is more
        consistent with :ref:`memoryviews` and :ref:`fusedtypes`. The C style
        declaration is soft-deprecated and it's recommended to use Java style
        declaration instead.

```

Reply:
Oh, there's a typo:

``` diff
-         .. code-blobk:: cython
+         .. code-block:: cython
```

Reply:
I like your suggestions but if you are not against, I would move it to separate sections